src/app/events/components/events-search/events-search.component.ts

```diff
@@ -26,7 +27,10 @@ import {EventRestService} from "../../../core/service/event/event-rest.service";
   styleUrl: './events-search.component.scss'
 })
 export class EventsSearchComponent {
-  eventService: EventService = inject(EventRestService);
+  @Output() eventSelected = new EventEmitter<Event>();
+
+  eventService: EventService = inject(EventMockService);
+  //eventService: EventService = inject(EventRestService);
```

Comment:
Przed mergem trzeba będzie zmienić na EventRestService

Reply:
Szkoda że nie ma jakiegoś mechanizmu podobnego do tych env ze jak jest odpalane w debug to jest mock a jak w release to rest

Reply:
Moze zostawmy mocki na developie na masterze damy normalne serwisy ?